chef/data_bags/crowbar/template-crowbar.json

```diff
@@ -25,17 +25,24 @@
     "crowbar": {
       "crowbar-revision": 10,
       "crowbar-applied": false,
+      "schema-revision": 11,
```

Comment:
OK, this is confusing. Should I bump `"crowbar-revision"`, or use new `"schema-revision"` here? There currently is 010 migrate file...